CONTRIBUTING.md

```diff
@@ -77,7 +77,7 @@ To skip P tests:
 
 ```powershell
 .PARAMETER CI
-  Exits after run.  Enables test results and code coverage on `/src/*`. Enable exit with 1 if tests don't pass. Forces P Tests to fail when `dt` is left in the tests. `dt` only runs the specified test, so leaving it in code would run only one test from the file on the server.
+  Exits after run. Enables test results and code coverage on `/src/*`. Enable exit with 1 if tests don't pass. Forces P Tests to fail when `dt` is left in the tests. `dt` only runs the specified test, so leaving it in code would run only one test from the file on the server.
```

Comment:
> Enable exit with 1 if tests don't pass

This is just me nitpicking, but failed P-tests returns 1. Failed tests during the normal Pester tests that run after will result in a value greater than zero (sum of failed tests, blocks++). Maybe "_Enables exit with non-zero exit code if tests don't pass._" ?